### ISSUE TITLE
Angular is deprecated - please migrate to React

### ISSUE THREAD

**Issue by tolzhabayev** [flant/grafana-statusmap#302]
# Hello :wave: from Grafana Labs

We have detected that your plugin might still be using `AngularJS` and recommend migrating it to `React`, so that it may continue to operate successfully in future versions of Grafana.

You can read more on why and how by visiting our documentation: https://grafana.com/docs/grafana/latest/developers/angular_deprecation/

You can reach out to us with any questions via slack: [slack.grafana.com](https://slack.grafana.com) - channel `#plugins` or our [plugin development community forum](https://community.grafana.com/c/plugin-development/30)

**Comment by biohazardxxx** [comment]
HI @diafour What are the chances that this plugin will be migrated? Thanks for your efforts 👍

**Comment by allanian** [comment]
+1

**Comment by Madaspe** [comment]
+1

**Comment by andreadna** [comment]
+1

**Comment by swissalpsboy** [comment]
+1
Thanks for your efforts

**Comment by mwiehler** [comment]
+1

**Comment by sinankizar** [comment]
I think this project is dead but there is no harm to bump this request :)

**Comment by DirkStr** [comment]
does somebody have a hint, which panel to use instead of this one?
or has already made experiences using a different one?

thanks!

**Comment by mwiehler** [comment]
@DirkStr I've migrated to grafanas' "Status history" panel which seems like a decent alternative.

**Comment by DirkStr** [comment]
@mwiehler : yes - also just tried it - seems to be suitbable. thanks for your fast response!!

**Comment by strowi** [comment]
Thx for the info abput "status history" look good, but for our case it is still missing a scrollbar or some form of pagination since we have quite a lot of metrics to display.

**Comment by timur-tukaev** [comment]
@nerddtvg @mwiehler Thanks for sharing these alternatives! And yes, this project is actually deprecated because of appearence of built-in library.
